### PR TITLE
fix(e2e): remove sequence number checks

### PR DESCRIPTION
**Pull request by czarcas7ic**
Closes: #XXX

## What is the purpose of the change

This PR is very exciting. Skip made a suggestion of removing sequence number check ante handler to prevent the issues that have been plaguing our E2E test suite. It appears to work!

### CODE REVIEW

File: app/ante_no_seq.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
+//go:build excludeIncrement
```
Comment (czarcas7ic) [inline app/ante_no_seq.go:1]: This ante handler only build with the excludeIncrement build flag, which will only be used in E2E

File: app/ante_no_seq.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	v9 "github.com/osmosis-labs/osmosis/v21/app/upgrades/v9"
+
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+
+	txfeeskeeper "github.com/osmosis-labs/osmosis/v21/x/txfees/keeper"
+	txfeestypes "github.com/osmosis-labs/osmosis/v21/x/txfees/types"
+)
+
+// Link to default ante handler used by cosmos sdk:
+// https://github.com/cosmos/cosmos-sdk/blob/v0.43.0/x/auth/ante/ante.go#L41
```
Comment (p0mvn) [inline app/ante_no_seq.go:27]:
```suggestion
// https://github.com/cosmos/cosmos-sdk/blob/v0.43.0/x/auth/ante/ante.go#L41
// N.B. There is a sister file called `ante_eq.go` that is used for production.
// Opposite to this, it also chains `IncrementSequenceDecorator` which is not needed for e2e testing but is critical for production.
// If you make a change here, make sure to make the same change in `ante.go`.
```

File: tests/e2e/containers/containers.go [hunk 3]
```diff
@@ -115,6 +117,16 @@ func (m *Manager) ExecTxCmdWithSuccessString(t *testing.T, chainId string, conta
...
 	}
+	// Add memo field to every tx
+	// This is done because in E2E, we remove the sequence number ante handler.
+	// This allows for quick throughput of txs, but if two txs are the same (i.e. two CL claims, two bank sends with the same amount),
+	// the sdk treats them as the same tx and errors with code 19 (tx already in mempool). By changing the memo field on every tx,
+	// we ensure that every tx is unique and can be submitted.
+	memo := fmt.Sprintf("--note=%d", memoCounter)
+	allTxArgs = append(allTxArgs, memo)
+	// Increment the counter for the next tx
+	memoCounter++
```
Comment (p0mvn) [inline tests/e2e/containers/containers.go:128]: I thought we can run txs in parallel? Do we not need to keep `memoCounter` behind mutex?

**Review by p0mvn — APPROVED** [review]
🚀

File: tests/e2e/containers/containers.go (hunk 3 above)
Comment (czarcas7ic) [reply tests/e2e/containers/containers.go:128]: So in order for this to be an issue, the tx happening in parallel would need to be the exact same tx, which is super unlikely (i.e. we would need to have have two separate tests call the same message with the same account at the same time). You are totally right though that it's possible and should probably be mutex'd, im just explaining the probable reason of why I haven't seen any failures yet.

Comment (czarcas7ic) [reply tests/e2e/containers/containers.go]: Done here https://github.com/osmosis-labs/osmosis/pull/7048/commits/614e81c6651cef5bb264f4b0c9de4491633d4847

**Pull request merged**